components/brave_webtorrent/browser/webtorrent_util.cc

```diff
@@ -57,6 +58,17 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(kWebTorrentEnabled, true);
 }
 
+bool IsWebtorrentPage(const GURL& url) {
```

Comment:
nit IsWebtorrentURL

Reply:
Addressed in https://github.com/brave/brave-core/pull/6085/commits/2a601f7357e113e9199e26cad2f33e930083ae5c